vector/src/main/java/im/vector/app/features/home/room/typing/TypingHelper.kt

```diff
@@ -51,9 +51,8 @@ class TypingHelper @Inject constructor(private val stringProvider: StringProvide
         }
     }
 
-<<<<<<< HEAD
     private fun getDisplayName(name: String) = TchapUtils.getNameFromDisplayName(name)
-=======
+
     fun getNotificationTypingMessage(typingUsers: List<SenderInfo>): String {
```

Comment:
Not for this PR but should we have a specific tchap code for DM in that place?

Reply:
For which difference do you want it ?

Reply:
i think this should be done in another PR if we want to update it.